azurerm/data_source_image.go

```diff
@@ -40,6 +40,11 @@ func dataSourceArmImage() *schema.Resource {
 
 			"location": locationForDataSourceSchema(),
 
+			"zone_resilient": {
```

Comment:
could we set this field in the state here: https://github.com/terraform-providers/terraform-provider-azurerm/pull/3100/files#diff-21521ec4fe0d38353d87860312cd25a4R187

Reply:
Good catch thanks!